.github/workflows/echo.yml

```diff
@@ -4,9 +4,19 @@ on:
   push:
     branches:
       - master
+    paths:
+      - '**.go'
+      - 'go.*'
+      - '_fixture/**'
+      - '.github/**'
```

Comment:
I added go.* so go.sum is also included. I also added .github/** so that it workflows are run when they have changed.